packages/office-ui-fabric-react/src/components/Nav/Nav.tsx

```diff
@@ -60,16 +68,15 @@ export class Nav extends BaseComponent<INavProps, INavState> implements INav {
       return null;
     }
 
-    // when this.props.groups[x].name is specified or Any of the link has child link, chevorn Expand/collaps button is shown,
-    // different padding is needed. _hasExpandButton marks this condition.
-    this._hasExpandButton = groups.some((group: INavLinkGroup, groupIndex: number) => {
-      return !!group && !!group.name || group.links && group.links.some((link: INavLink, linkIndex: number) => {
-        return !!link && !!link.links && link.links.length > 0;
-      });
+    // When groups[x].name is specified or any of the links have children, the expand/collapse
+    // chevron button is shown and different padding is needed. _hasExpandButton marks this condition.
+    this._hasExpandButton = groups.some((group: INavLinkGroup) => {
+      return group ? !!group.name || group.links && group.links.some((link: INavLink) => {
```

Comment:
Nit: I would throw a set of parenthesis around the second part of the or statement, just so it's more clear that they're together.